SkatkovAM/lab_2/lab2.cpp

```diff
@@ -0,0 +1,109 @@
+
+#include <iostream>
+
+#include <ctime>
+using namespace std;
+void basicArrayInit(int * arr, int size, int max){
+    for(int i=0; i<size;i++){
+        arr[i] = rand() % max+ 1;
+    }
+}
+void oddArrayInit(const int * basic_arr,int *odd_arr, const int size, int max){
+    int odd_arr_ind=0;
+    odd_arr[odd_arr_ind]=max+1;//unreal value for basic_array[i]  to check later
+    for(int i=0;i<size;i++){
+        odd_arr[i+1 - (i==size-1)]=max+1; // weird index to not to init odd_arr[size]
+        if (basic_arr[i]%2){
+        odd_arr[odd_arr_ind]=basic_arr[i];
+            odd_arr_ind++;
+
+        }
+
+    }
+}
+
+void sortAnyArray(int * arr,const int size){  //bubbly
+    int im_helpful;
+    for(int j=0;j<size;j++) {
+        for (int i = 0; i < size-1; i++) {
+            if(arr[i]<arr[i+1]){
+                im_helpful=arr[i];
```

Comment:
You can define and init im_helpful at the same line

Reply:
does it makes sense, when I init it in each cycle of for later?